src/Entity/SettingInterface.php

```diff
@@ -109,6 +109,17 @@ public function getSettingType(): string;
      */
     public function hasInitialization(): bool;
 
+    /**
+     * Specification:
+     * - Returns `true` if a value should be set by a user in any case.
+     * - Returns `false` for default behavior.
+     *
+     * @api
+     *
+     * @return bool
+     */
+    public function isForceAskValue(): bool;
```

Comment:
maybe isRequire() ? if the setting is required we need to ask it.

Reply:
It was discussed with @pavelmaksimov25. Actually we have all the settings required. If you set null value in settings.yaml this value will be asked in any cases. But if you set some default values the question will be skipped.